openstack_hypervisor/cli/pci_devices.py

```diff
@@ -332,6 +373,22 @@ def to_output_schema(snap: Snap, nics: list[Interface]) -> NicList:  # noqa: C90
     return NicList(nics_)
 
 
+def to_output_schema_gpus(snap: Snap) -> GpuList:  # noqa: C901
+    """Convert the gpus to the output schema."""
+    gpus_ = []
+
+    pci_spec_cfg = _get_pci_spec_cfg(snap)
+    for pci_device in pci.list_pci_devices():
+        if not pci.is_display_device(pci_device["class"]):
```

Comment:
something I'm aware of, but I can't find a reference online, it's that AFAIK there are server-grade GPUs that have no display output, what would be the behavior in a situation like that?

Reply:
Even I am not sure what should be considered as GPU devices. So I modified to include Display controllers 0x03 and Processor accelerators 0x1200 [1] as GPU devices

[1] https://admin.pci-ids.ucw.cz/read/PD